tests/end-to-end/backup/chart/backup-test/templates/rbac.yaml

```diff
@@ -72,6 +69,15 @@ rules:
 - apiGroups: ["gateway.kyma-project.io"]
   resources: ["apirules"]
   verbs: ["create", "delete", "get", "list"]
+- apiGroups: ["messaging.knative.dev"]
+  resources: ["channels","channels/status"]
+  verbs: ["*"]
+- apiGroups: ["serving.knative.dev"]
+  resources: ["services", "services/status"]
+  verbs: ["*"]
```

Comment:
is the test creating knative channels and ksvcs???? if no then, does it makes sense to give all the powers in this clusterrole?

Reply:
removing unnecessary rbac verbs is next on my list (after making sure it works :D )

Reply:
now it is